tsconfig.json

```diff
@@ -8,7 +8,9 @@
     "noImplicitOverride": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "alwaysStrict": true,
```

Comment:
This option defaults to `true` since we have `strict` set to `true`, so it's not needed here.  https://www.typescriptlang.org/tsconfig#alwaysStrict

Reply:
Are you referring to noFallthroughCasesInSwitch not being necessary or alwaysStrict?

Reply:
`alwaysStrict`